app/build.gradle

```diff
@@ -180,7 +179,11 @@ dependencies {
       'org.jetbrains.kotlinx:kotlinx-coroutines-core:1.4.1',
       'org.jetbrains.kotlinx:kotlinx-coroutines-android:1.4.1',
       'org.mockito:mockito-core:2.7.22',
-      'com.github.oppia:android-spotlight:ebde38335bfb56349eae57e705b611ead9addb15'
+      'com.github.oppia:android-spotlight:ebde38335bfb56349eae57e705b611ead9addb15',
+
+      //Sambhrant Tiwari
+      'com.intuit.ssp:ssp-android:1.0.5',
+     'com.intuit.sdp:sdp-android:1.0.5'
```

Comment:
I think adding the sdp lib only is sufficient, and there is a newer version, 'com.intuit.sdp:sdp-android:1.1.0',